aion.sh

```diff
@@ -46,7 +46,7 @@ fi
 
 # to suppress illegal reflective access warning out of xnio and protobuf
 # (we depend on xnio transitively via undertow-core)
-JAVA_OPTS+=" --add-opens=java.base/sun.nio.ch=ALL-UNNAMED --add-opens=java.base/java.nio=ALL-UNNAMED"
+JAVA_OPTS+=" -Xmx8g --add-opens=java.base/sun.nio.ch=ALL-UNNAMED --add-opens=java.base/java.nio=ALL-UNNAMED"
```

Comment:
Would it make more sense to add this on line 44 instead?  That's where Xms is set.

Reply:
updated!